src/sentry/models/signals.py

```diff
@@ -8,5 +8,6 @@
 
 @receiver(post_save, sender=Group)
 def resource_changed(sender, instance, created, **kwargs):
-    from sentry.tasks.servicehooks import process_resource_change
-    process_resource_change.delay(sender, instance.id, created)
+    if created:
+        from sentry.tasks.servicehooks import process_resource_change
+        process_resource_change.delay(sender, instance.id, created)
```

Comment:
Should you also remove the updated forks from this task method?

Reply:
Yea, probably, good call.